### PR TITLE
Update RTS examples for 3.5.0

### PR DESCRIPTION
**Pull request by jmkiley**
Switched `MGLStyleFunction`s to `MGLCameraStyleFunction`.

Did not update these examples to use DDS.

### CODE REVIEW

File: Examples/ObjectiveC/RuntimeAddLineExample.m [hunk 1]
```diff
@@ -52,10 +52,12 @@ - (void)drawPolyline:(NSData *)geoJson {
     layer.lineCap = [MGLStyleValue valueWithRawValue:[NSValue valueWithMGLLineCap:MGLLineCapRound]];
     layer.lineColor = [MGLStyleValue valueWithRawValue:[UIColor colorWithRed:59/255.0 green:178/255.0 blue:208/255.0 alpha:1]];
     // Use a style function to smoothly adjust the line width from 2pt to 20pt between zoom levels 14 and 18. The `interpolationBase` parameter allows the values to interpolate along an exponential curve.
-    layer.lineWidth = [MGLStyleValue valueWithInterpolationBase:1.5 stops:@{
-        @14: [MGLStyleValue valueWithRawValue: @2],
-        @18: [MGLStyleValue valueWithRawValue: @20]
-    }];
+    layer.lineWidth = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeExponential
+        cameraStops:@{
+                      @14: [MGLStyleValue valueWithRawValue: @2],
```
Comment (friedbunny) [inline Examples/ObjectiveC/RuntimeAddLineExample.m]: Let’s keep the indentation the same as it was (i.e., more to the same) — may prevent line-wrapping issues.

File: Examples/ObjectiveC/RuntimeAddLineExample.m [hunk 2]
```diff
@@ -52,10 +52,12 @@ - (void)drawPolyline:(NSData *)geoJson {
...
+                      @18: [MGLStyleValue valueWithRawValue: @20]
+                      }
+        options:@{MGLStyleFunctionOptionDefaultValue: @1.5}];
```
Comment (friedbunny) [inline Examples/ObjectiveC/RuntimeAddLineExample.m]: Add a space after `options:`.

File: Examples/ObjectiveC/RuntimeMultipleAnnotationsExample.m [hunk 3]
```diff
@@ -49,32 +49,40 @@ - (void)addItemsToMap:(NSArray *)features {
...
-        @2: [MGLStyleValue valueWithRawValue:@0.5],
-        @7: [MGLStyleValue valueWithRawValue:@1.0],
-    }];
-    circles.circleRadius = [MGLStyleValue valueWithStops:@{
-        @2: [MGLStyleValue valueWithRawValue:@2],
-        @7: [MGLStyleValue valueWithRawValue:@3],
-    }];
-
+    circles.circleOpacity = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeExponential
+        cameraStops : @{
```
Comment (friedbunny) [inline Examples/ObjectiveC/RuntimeMultipleAnnotationsExample.m]: Remove the space after `cameraStops`.

File: Examples/ObjectiveC/RuntimeMultipleAnnotationsExample.m [hunk 4]
```diff
@@ -49,32 +49,40 @@ - (void)addItemsToMap:(NSArray *)features {
...
+                        @2: [MGLStyleValue valueWithRawValue:@0.5],
+                        @7: [MGLStyleValue valueWithRawValue:@1.0]
+                        }
+        options: @{MGLStyleFunctionOptionDefaultValue : [MGLStyleValue valueWithRawValue:@0.75]}];
```
Comment (friedbunny) [inline Examples/ObjectiveC/RuntimeMultipleAnnotationsExample.m]: Remove the space after `MGLStyleFunctionOptionDefaultValue`.

File: Examples/ObjectiveC/RuntimeMultipleAnnotationsExample.m [hunk 5]
```diff
@@ -49,32 +49,40 @@ - (void)addItemsToMap:(NSArray *)features {
...
+                             
+    circles.circleRadius = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeInterval cameraStops:@{
```
Comment (friedbunny) [inline Examples/ObjectiveC/RuntimeMultipleAnnotationsExample.m]: I suspect a line this long will cause it to wrap on our website. Either way, let’s keep the line breaks for `cameraStops` consistent.

File: Examples/ObjectiveC/RuntimeAddLineExample.m [hunk 6]
```diff
@@ -52,10 +52,12 @@ - (void)drawPolyline:(NSData *)geoJson {
...
     // Use a style function to smoothly adjust the line width from 2pt to 20pt between zoom levels 14 and 18. The `interpolationBase` parameter allows the values to interpolate along an exponential curve.
-    layer.lineWidth = [MGLStyleValue valueWithInterpolationBase:1.5 stops:@{
-        @14: [MGLStyleValue valueWithRawValue: @2],
-        @18: [MGLStyleValue valueWithRawValue: @20]
-    }];
+    layer.lineWidth = [MGLStyleValue valueWithInterpolationMode:MGLInterpolationModeExponential
+        cameraStops:@{
+            @14: [MGLStyleValue valueWithRawValue:@2],
+            @18: [MGLStyleValue valueWithRawValue:@20]
+            }
```
Comment (friedbunny) [inline Examples/ObjectiveC/RuntimeAddLineExample.m]: These closing braces should align with the start of `cameraStops`.

Comment (jmkiley) [inline Examples/ObjectiveC/RuntimeAddLineExample.m]: I went back and forth on that one, thanks :)

**Pull request merged**